### DISCUSSION TITLE
Customizing the admin console for Keycloak v24

### DISCUSSION THREAD

**Discussion by panda1846** [keycloak/keycloak#29075] [Q&A]
Hi,

In a project of mine, I want to customize the admin console and add a new tab in the users view that displays two buttons that send requests to endpoints. I also want to display a dialog that shows the content of the requests and buttons with further actions. Is this type of customization even possible with the way the admin console customization is implemented in the Keycloak quickstarts (https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-admin-console-spi)?

I would be very grateful if someone could help me and give me advice on how to implement my changes!

Thanks in advance

**Comment by jonkoops** [comment]
This is currently not possible, see #13598
